news/_posts/2020-08-12-write-product-specifications.md

```diff
@@ -61,19 +61,19 @@ When a new feature is suggested or planned, it is essential to define its scope
 
 ### How-to
 
-Now let us focus on the practical side! Starting to write a specification, you have two choices: the page already exists in the specifications repository and you just have to edit it to complete it, or it does not and then you will have to create it to suggest your specifications.
+Now let us focus on the practical side! Starting to write a specification, you have two choices: the page already exists in the specifications repository and you just have to edit it to complete it, or it does not exist yet and then you will have to create it to suggest your specifications.
 
 Whether it is an edition or addition, a pull request (PR) will have to be created - just like for any development! You will learn more about the detailed steps in the `CONTRIBUTING.md` file [here](https://github.com/PrestaShop/prestashop-specs/blob/master/CONTRIBUTING.md#process-to-follow) and the suggested template to use in the next section of this post.
 
 ![PrestaShop Product Specifications Management](/assets/images/2020/08/prestashop-specs-management.jpg)
 
-Once your specifications submitted, the pull request is reviewed by the core team and validated by the product team. If you specify a new feature, the process is slightly different: specifications will have to be validated before starting the development and will be merged when the development is done. Why? Just to make sure that all public specifications match the current features and not behaviors that are not available yet.
+Once your specifications have been submitted, the pull request is reviewed by the core team and validated by the product team. If you specify a new feature, the process is slightly different: specifications will have to be validated by the core team and merged before starting the development and will be merged when the development is done. If any behavior is redefined during the development phase, the specifications must be updated accordingly following the same workflow: PR > validation > merge > development.
```

Comment:
@eternoendless, @Progi1984, and @SimonGrn, I let you check the paragraph right here. ;-)

Reply:
I approve. You could add that specifications might go through QA too, but since it's not mandatory (and depending on the size of the modifications since we're limited in bandwidth).

Reply:
```suggestion
Once your specifications have been submitted, the pull request is reviewed by the core team and validated by the product team. If you specify a new feature, the process is slightly different: specifications will have to be validated by the core team (product, dev, and QA if need be) and merged before starting the development and will be merged when the development is done. If any behavior is redefined during the development phase, the specifications must be updated accordingly following the same workflow: PR > validation > merge > development.
```

Reply:
@SimonGrn, is it better now?

Reply:
Perfect :) !